src/backend/distributed/cdc/Makefile

```diff
@@ -1,15 +1,38 @@
-MODULE_big = citus_pgoutput
-EXTENSION = citus_pgoutput
+ifneq ($(DECODER),)
+    DECODER_NAME=$(DECODER)
+else
+	DECODER_NAME = pgoutput
```

Comment:
do we need a separate DECODER_NAME variable?

Reply:
Ok I have removed it now, just using DECODER